lms/static/sass/discussion/_mixins.scss

```diff
@@ -148,14 +146,13 @@
 }
 
 @mixin forum-user-label($color) {
-  @include font-size(9);
   @extend %t-weight5;
+  font-size: 9px;
```

Comment:
We should use one of our defined size variables, or use rem/em. Also, 9px is tiny.